docs/docs/composition-vs-inheritance.md

```diff
@@ -151,7 +151,7 @@ class SignUpDialog extends React.Component {
   }
 
   handleChange(e) {
-    this.setState({login: e.target.value});
+    this.setState(() => ({login: e.target.value}));
```

Comment:
Also this is all for a issue requesting the docs be updated

I just did this and the synthetic event is logged, as expected.  Is there a specific property that I can be checking on this event to determine whether its been recycled?

```
import React, {Component} from 'react'
export default class App extends Component {
  test = (se) => {
    console.log('test', se)
    this.setState(() => {
      console.log('setState', se)
      return {}
    })
  }

  render () {
    return (
      <div>
        <button onClick={this.test}>Test</button>
      </div>
    )
  }
}

```

Thanks!